hrm-service/service-tests/case-search.test.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from './case-validation';
 import { CaseListFilters, DateExistsCondition } from '../src/case/case-data-access';
 import * as contactDb from '../src/contact/contact-data-access';
-import * as proxiedEndpoints from './external-service-stubs/proxied-endpoints';
```

Comment:
It doesn't look like `./external-service-stubs/proxied-endpoints` has been deleted, but I can't find it being actively used anywhere. Should that file be deleted?